### ISSUE TITLE
Extranonce for nicehash

### ISSUE THREAD

**Issue by banjochose** [luke-jr/bfgminer#684]
Hello please add possibility to set extranonce 4+ .Currently it wont work with nicehash because they require be it least 4. Tried to add #xnsub after stratum port, but bfgminer gives error saying it's not port number

**Comment by emx0r** [comment]
is there any way how to increase it to 4?

**Comment by luke-jr** [comment]
Yes, by editing the code. But most upstream pools won't allow it.
